test/test.js

```diff
@@ -30,3 +30,26 @@ test('allows special characters in file paths', async t => {
 	await client.waitUntilTextExists('h1', '🚀', 5000);
 	t.pass();
 });
+
+test('fallbacks to root index if unresolved path has .html extension or no extension', async t => {
+	t.context.spectron = new Application({
+		path: electron,
+		args: ['fixture-dir-fallback.js']
+	});
+	await t.context.spectron.start();
+	const {client} = t.context.spectron;
+	await client.waitUntilWindowLoaded();
+	await client.waitUntilTextExists('h1', '🦄', 5000);
+	t.pass();
+});
+
+test('throws error if unresolved path has an extension other than .html', async t => {
+	t.context.spectron = new Application({
+		path: electron,
+		args: ['fixture-404-error.js']
+	});
+	await t.context.spectron.start();
+	const {client} = t.context.spectron;
+	await client.waitUntilWindowLoaded();
+	await t.throwsAsync(client.waitUntilTextExists('h1', '🦄', 5000));
```

Comment:
You used the assertion incorrectly, and when fixed (https://github.com/sindresorhus/electron-serve/pull/13/commits/037261161e8e2a07fe0cfa3214bde415d25bede0), it no longer passes.

Reply:
Upgrading `ava` to v2 fixed the issue and the test cases are passing.

I don't know if it is out of scope of this PR.

Reply:
That's fine :)